marlowe-runtime/config/Language/Marlowe/Runtime/CLI/Option.hs

```diff
@@ -49,6 +49,14 @@ syncHeaderPort =
     3725
     "The port number of the marlowe-sync server's header synchronization API."
 
+syncBulkPort :: CliOption OptionFields PortNumber
+syncBulkPort =
+  port
+    "marlowe-bulk"
+    "SYNC_MARLOWE_BULK"
+    3730
+    "The port number of the marlowe-sync server's bulk synchronization API."
```

Comment:
Won't the operables, deployment, and compose derivations need updating, too?

In fact, should we add this reminder to the PR template?

Reply:
That might be a good way to deal with this. You're right we often forget this. I can't really think of a good way to automate this check.